### DISCUSSION TITLE
Net electric output efficiency

### DISCUSSION THREAD

**Discussion by jonmaddock** [ukaea/PROCESS#3239] [Ideas]
In GitLab by @mkovari on Mar 4, 2019, 10:54

Design and analysis of the improved configuration of the secondary circuit for the EU-DEMO power plant

https://www.sciencedirect.com/science/article/pii/S0920379619301693

This paper gives a single operating point for a WCLL DEMO with (huge) molten salt tanks for energy storage.  If we wanted to we could use these results to recalibrate the PROCESS balance of plant module.

@hlux @stuartmuldrew @jmorris-uk @ajpearcey

**Comment by chris-ashe** [comment]
The paper presents a energy storage system and secondary loop to be used in conjunction when in the dwell phase. The is meant to allow constant electricity to be produced when in dwell. PROCESS does not currently have models to calculate this and does not have the in-depth systems models that the paper tries to implement. This paper should be revisited if the power conversion models in PROCESS are much higher fidelity.
